src/components/calcite-popover/calcite-popover.scss

```diff
@@ -3,12 +3,21 @@ $image_max_height: 200px;
 @include popperHost();
 @include popperArrow();
 
+:host {
+  pointer-events: none;
+}
+
+:host-context([open]) {
```

Comment:
I believe this isn't well supported in Firefox nor Safari: https://caniuse.com/?search=host-context

Reply:
Thanks. This was previously there and also is used in many other places so we should probably have a new issue for this one. https://github.com/Esri/calcite-components/search?l=SCSS&q=host-context

Reply:
We might just be using host-context incorrectly and can just replace with host in these cases. What do you think?

Reply:
If `host` works for those cases, it sounds like a good solution. 👍

Reply:
@jcfranco should we create an issue to remove all host-context?

Reply:
@paulcpederson seems like the host-context was added here: https://github.com/Esri/calcite-components/commit/41413198cbab1bc291d41e46a1fc48e31a32b8f3#diff-f22f03c93da9eb6e0482837d3139c65705c36c4a244913514a1feb1e04a6ebce

Reply:
I'll fix the popper file to also have host as well as host-context for RTL. Can one of you create an issue to fix the other host-context in the project?

Reply:
Created issue.